src/reducers/search.js

```diff
@@ -34,6 +34,11 @@ import union from 'lodash.union';
 import uniqBy from 'lodash.uniqby';
 
 export const initialState = {
+  defaultTag: {
+    displayName: 'Top inspiration',
+    colour: '#8EB8C4',
```

Comment:
The colour is [determined by the tag type](https://github.com/numo-labs/isearch-ui/blob/master/lib/tags/index.js#L30), so doesn't need to be passed.